src/common/base/Status.h

```diff
@@ -81,7 +81,8 @@ class Status final {
         return Status(k##ERROR, msg);                   \
     }                                                   \
                                                         \
-    static Status ERROR(const char *fmt, ...) {         \
+    static Status ERROR(const char *fmt, ...)           \
+        __attribute__((format(printf, 1, 2))) {         \
```

Comment:
Consider `FOLLY_PRINTF_FORMAT_ATTR`?

Reply:
Thanks, I'm deliberately not relying on folly